docs/modules/ROOT/pages/module/code.adoc

```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
 }
 ----
 
-[[settings]]
-== Settings
+[[assets]]
+== Assets
+
+The Assets page allows you to manage your Code Inspector assets, supporting two types: Smart Contracts and GitHub repositories. When you install the Code Inspector GitHub app and select repositories, they will appear on this page. Additionally, you can create Smart Contract assets from addresses in your address book. Note that Smart Contract assets must be verified on Etherscan for this feature to work.
+
+Smart Contract assets can be manually triggered for reports on the report page. GitHub assets can be triggered manually and automatically from new pull requests (PRs) and commits.
+
+image::code-assets.png[Code Assets]
+
+[[asset-settings]]
+=== Settings
+
+The Asset Settings page offers two main settings to manage how Defender handles asset security: Automatically Generate Report and Vulnerability Detection.
+
+==== Automatically Generate Report
 
-The Settings page allows you to manage the permissions and access level of the Code Inspector.
+Automatically Generate Report, is specifically for GitHub repositories. When enabled, this setting ensures continuous monitoring by generating new reports automatically whenever a new pull request (PR) is opened or a new commit is pushed to a PR. 
 
-In the Repositories tab, you can view all the repositories where the app is currently installed, helping you track where the app is actively conducting code analysis. If you need to make changes to the repositories that the app has access to, a convenient link takes you directly to the GitHub settings page of the app, facilitating effortless repository management.
+==== Vulnerability Detection
+
+Vulnerability Detection provides a robust security measure by automatically scanning your assets for new critical vulnerabilities as they are discovered. This setting is applicable to both GitHub repositories and smart contracts. To enable this feature, users must activate the setting and select a specific branch to be monitored for GitHub assets. Key features of Vulnerability Detection include:
```

Comment:
"Active Vulnerability Detection and Notification provides a robust security measure by automatically scanning your assets for new vulnerabilities. 
This setting is applicable to both GitHub repositories and smart contracts. To enable this feature for Github repositories assets, you must activate the setting and select a specific branch to be monitored."